docs/examples/minimal-class.py

```diff
@@ -0,0 +1,9 @@
+from r2t2 import add_reference
```

Comment:
minor suggestion: maybe it would be good to keep the underscore (`_`) for Python files?

Reply:
The `...` will fail in any test and when executing. This is why I removed the `...`.

Reply:
I would just not use `...` or convert it to a comment (e.g. `# ...`).